setup.py

```diff
@@ -22,7 +22,7 @@
     # Versions should comply with PEP440.  For a discussion on single-sourcing
     # the version across setup.py and the project code, see
     # https://packaging.python.org/en/latest/single_source_version.html
-    version='0.1.2',
+    version='0.1.3',
```

Comment:
Unsure if the version bump should go in this PR or a separate one, since there may be a couple other smaller PRs as well.

Reply:
What is the policy of changing the version? This change looks like a substantial one to justify bumping the version.

Reply:
We usually bump version in separate follow up PRs, can you first check this in then create a follow up one?

Reply:
Sure.